tests/data/tasks/xlsx/test-xlsx-to-component-definition.config

```diff
@@ -0,0 +1,14 @@
+[task.xlsx-to-oscal-component-definition]
+
+catalog-file = nist-content/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.json
+spread-sheet-file = tests/data/spread-sheet/good.xlsx
+work-sheet-name = example_best_practices_controls
+output-dir = tests/data/tasks/xlsx/output
+output-overwrite = true
+
+org-name = International Business Machines
+org-remarks = IBM
+namespace = http://ibm.github.io/compliance-trestle/schemas/oscal/cd/ibm-cloud
+property-name-to-class = goal_name_id:scc_goal_name_id, goal_version:scc_goal_version
+catalog-url = https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev4/json/NIST_SP-800-53_rev4_catalog.json
```

Comment:
No biggy, but is this the best config format to use?  I guess you are using configparser and a fairly simple format.  There are other options so if config files like this might be needed for future, more complex, tasks it would be good to make a good choice.  But this may be fine.

Reply:
I think there is an open issue for centralising the configuration parsing which is a little spread out right now. Today - we should part this.